build.sbt

```diff
@@ -15,8 +15,8 @@ lazy val core = module("core", hideFolder = true)
     flags    = "cats" :: Nil,
     yaxScala = true))
   .crossDepSettings(
-    %%("cats-core"),
```

Comment:
This can be done in this way ` %%("cats-core", "1.0.0")` too ;)

Reply:
Fixed :).